homeassistant/components/snooz/fan.py

```diff
@@ -71,15 +72,18 @@ async def async_setup_entry(
 class SnoozFan(FanEntity, RestoreEntity):
     """Fan representation of a Snooz device."""
 
+    _attr_has_entity_name = True
+    _attr_name = None
+
     def __init__(self, data: SnoozConfigurationData) -> None:
         """Initialize a Snooz fan entity."""
         self._device = data.device
-        self._attr_name = data.title
```

Comment:
Do we need to clean up the model of this data object as well? The title doesn't seem to be used elsewhere?

Reply:
The title is used in the unload/reload function in `__init__.py`

Reply:
Ooh I see now, that is an odd construct, but ok 😄 

../Frenck